from app.domain.shared.exception import DomainError


class ConnectionLinkNotBelongsToSocialNetworkError(DomainError):
    pass
